### ISSUE TITLE
Debian 11 dependencies

### ISSUE THREAD

**Issue by r13s** [lutris/lutris#4625]
Greetings.

Can’t install `0.5.12` because package `gir1.2-webkit2-4.1` is only available in `testing`, `unstable` and `experimental`, but not in `stable` or even `backports`.

On the other hand, the `libwine-development` package only exists in `oldoldstable`, `oldstable` and `unstable`. Instead, there is the `libwine-dev` package.

As a result, `0.5.12` is broken in `stable` Debian 11.

Thank you.

**Comment by danieljohnson2** [comment]
So, Lutris 0.5.12 would work in Debian 11 unstable right?

Did 0.5.11 work in Debian 11 properly? There's been one change- from webkit 4.0 to 4.1.  This was done for compatibility with the next GNOME coming soon, but Lutris will actually fall back to 4.0 if 4.1 is not available.

I'm no .deb wizard, but maybe there's a way to say "requires 4.0 or 4.1, prefers 4.1?"

**Comment by strycore** [comment]
I've reverted to webkit2-4.0, the change to 4.1 should have never happened.

**Comment by strycore** [comment]
Lutris doesn't need libwine, it only need the dependencies needed by libwine. Any version should be fine.

**Comment by strycore** [comment]
I've made the dependency satisfied for both libwine-development and libwine. The deb available on Github and the one available on OBS reflect this change. 
Those should be installable on Debian

**Comment by AndrewAmmerlaan** [comment]
> I've reverted to webkit2-4.0, the change to 4.1 should have never happened.

Why was this reverted completely? Would it be possible to keep the part of the commit that allows using either version 4.0 or 4.1, without changing the default version? (https://bugs.gentoo.org/888669)

**Comment by hjoukl** [comment]
> The deb available on Github and the one available on OBS reflect this change. 
> Those should be installable on Debian

Thanks a lot - I can also confirm these are installable on my (K)ubuntu 20.04, fwiw. I was wondering a bit since the releases entry https://github.com/lutris/lutris/releases/tag/v0.5.12 is actually older than this issue but obviously the attached 0.5.12 deb file was fixed/changed afterwards.

As of yesterday the Ubuntu Lutris PPA (ppa:lutris-team/lutris) installation didn't work for me, due to the unsatisfiable gir1.2-webkit2-4.1 dependency.

**Comment by Vaccano** [comment]
It seems like some people who know what they are doing have figured out a way around this issue.  Any chance that someone would be willing to post some steps?

Basically, I see two parts that need doing.  First, the missing gir1.2-webkit2-4.1 needs to be fixed.  Some have mentioned going back to version 4.0, but no indication of how to do that was mentioned.

The second issue is bypassing all the missing recommends.  The option `--no-install-recommends` seems to help with that.  assuming it does not cause failures down the line.

If anyone who is an expert in these could offer more information, that would be super helpful!  (I am also open to other workarounds that will allow me to install lutris.)

**Comment by hjoukl** [comment]
> It seems like some people who know what they are doing have figured out a way around this issue. Any chance that someone would be willing to post some steps?

Not to say that I know what I'm doing. ;-)

What worked for me: Installing https://github.com/lutris/lutris/releases/tag/v0.5.12 *directly* i.e. download from there, install the local, downloaded `.deb` (like in `apt install ./Downloads/lutris_0.5.12_all.deb`). 

The mismatched dependencies have been addressed there. What did not work at the time: Install from the "official" Ubuntu Lutris PPA (ppa:lutris-team/lutris) that I added to my apt repo sources.
